Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -2857,7 +2857,7 @@ module.exports = {
    *```javascript
    *  onScroll={Animated.event(
    *    [{nativeEvent: {contentOffset: {x: this._scrollX}}}],
-   *    {listener},          // Optional async listener
+   *    {listener: (event, gestureState) => console.log(gestureState)}, // Optional async listener
```

Comment:
I trust your opinion, but gestureState is passed along as the second param... so isn't it better to document it?